en/debug/system_console.md

```diff
@@ -102,53 +121,9 @@ Then select 'serial connection' and set the port parameters to:
 * 1 stop bit
 
 
-## MAVLink Shell {#mavlink_shell}
-
-For NuttX-based systems (Pixhawk, Pixracer, ...), the *nsh console* can also be accessed via MAVLink over serial (USB/Telemetry) or WiFi (UDP/TCP) links.
-
-The easiest way to access the *nsh console* via MAVLink is using [QGroundControl](https://docs.qgroundcontrol.com/en/analyze_view/mavlink_console.html) (see **Analyze View > Mavlink Console**).
-
-You can also access the shell in a terminal using the **mavlink_shell.py** script:
-1. Shut down QGroundControl.
-1. Install dependencies:
-   ```sh
-   sudo pip3 install pymavlink pyserial
-   ```
-1. Open terminal (in Firmware directory) and start the shell:
-   ```sh
-   # For serial port
-   ./Tools/mavlink_shell.py /dev/ttyACM0
-   ```
-    ```sh
-   # For Wifi connection
-   ./Tools/mavlink_shell.py 0.0.0.0:14550
-   ```
-
-Use `mavlink_shell.py -h` to get a description of all available arguments.
-
-
-## Getting Started on the Console/Shell {#getting-started-on-the-console}
+### Snapdragon DSP Console
```

Comment:
So this is I think how you access the PX4 console. Ie you can't just use the terminal app out of the box but have to run some other stuff first. Is that correct?

Because this is different, I proposed to also split this out into snapdragon setup docs too. OK?

Reply:
Yes this is a lot different on Snapdragon and very specific to that platform, so splitting it out makes sense.